src/app.ts

```diff
@@ -83,6 +85,9 @@ function registerRoutes(app: express.Application, appEnvIsProduction: boolean) {
   app.use(resetPasswordCheckEmailRouter);
   app.use(checkYourEmailRouter);
   app.use(createPasswordRouter);
+  if (supportMFAOptions()) {
+    app.use(multiFactorAuthenticationRouter);
```

Comment:
i would think about the name of this component as mfa doesn't really mean much. It is more about selecting your mfa option. Some ideas:
select-mfa-options
choose-mfa-option